openrr-apps/src/bin/robot_command.rs

```diff
@@ -128,4 +130,56 @@ mod tests {
     fn assert_app() {
         RobotCommandArgs::command().debug_assert();
     }
+
+    #[test]
+    fn assert_sample_command() {
+        let bin = env!("CARGO_BIN_NAME");
+        let manifest_dir = Path::new(env!("CARGO_MANIFEST_DIR"));
+
+        let config_path = manifest_dir.join("config/sample_robot_client_for_urdf_viz.toml");
+        let config_path_arg = "--config-path=".to_string() + config_path.to_str().unwrap();
+        let sample_command_path = manifest_dir.join("command/sample_cmd_urdf_viz.txt");
```

Comment:
"command/sample_cmd_urdf_viz.txt" is repeated in L159.
I think it should be defined as const value.

Reply:
Thank you for your attention.